app/src/lib/emails/ForgotPasswordTemplate.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable i18next/no-literal-string */
+import React from "react";
```

Comment:
You can remove the eslint-disable comments at the tops of these files now. They disable the rule that everything needs to be translated, which you've now fixed :tada: